### ISSUE TITLE
No space after the bullet in items of a latex list produced by Sphinx

### ISSUE THREAD

**Issue by kwankyu** [sphinx-doc/sphinx#3664]
Subject: In the pdf document of the reference manual of the SageMath project, items of a list do not have a space between the bullet and the content.

### Problem
The sample latex code below is simplified from the actual output by Sphinx from a docstring. If I compile the sample latex code, then the item in the list under INPUT: appears like

*abc

rather than the expected

* abc

The cause of this problem seems to lie in the `fulllineitems` environment, where `\labelsep` is set 0pt. I tested with Sphinx 1.5.5

#### Procedure to reproduce the problem
```
\def\sphinxdocclass{report}
\documentclass[letterpaper,10pt,english]{sphinxmanual}

\usepackage{babel}
\usepackage{sphinx}

\begin{document}

\begin{itemize}
\item {} abc
\end{itemize}

\begin{fulllineitems}
\pysiglinewithargsret{}{}{}
Construct a prime divisor.

INPUT:
\begin{itemize}
\item {} abc
\end{itemize}

\end{fulllineitems}

\end{document} 
```

**Comment by jfbu** [comment]
well, you are right about `fulllineitems`. This environment is, it seems, not thought out to allow sub-environments like `enumerate` or `itemize`. In order for `description` to be usable therein, it redefines it globally (!!) in particular for the `labelsep` setting. This looks very ad hoc.

Could you check the patch at #3665 ?

**Comment by kwankyu** [comment]
I checked that the patch at #3665 works well. Thank you for your prompt response!

**Comment by jfbu** [comment]
Thanks @kwankyu for the report and the check. Fixed at 7c2f5f1 (#3665), merged in branches for 1.5.6 and 1.6.
